symphony-app-authentication-fe/services/registerApplication.js

```diff
@@ -0,0 +1,23 @@
+import { Utils } from '../js/utils.service';
+
+const dependencies = [
+  'ui',
+  'extended-user-info',
+  'modules',
+  'entity',
+  'dialogs',
+];
+
+/*
+* registerApplication                       register application on symphony client
+* @params       config                      app settings
+* @return       SYMPHONY.remote.hello       returns a SYMPHONY remote hello service.
+*/
+export const registerApplication = (config, appData) => {
+  const controllerName = `${config.appId}:controller`;
+
+  return SYMPHONY.application.register(
+          appData, 
+          dependencies, 
+        ).then(register);
```

Comment:
There is no 'register' function here. The library consumer should have the ability to provide a custom callback function to execute after this call or you should return this promise  after finishing the whole process.

Reply:
Done.